source/config/configSpec.py

```diff
@@ -13,7 +13,7 @@
 #: provide an upgrade step (@see profileUpgradeSteps.py). An upgrade step does not need to be added when
 #: just adding a new element to (or removing from) the schema, only when old versions of the config
 #: (conforming to old schema versions) will not work correctly with the new schema.
-latestSchemaVersion = 11
+latestSchemaVersion = 12
```

Comment:
Just one additional thing I thought of.
https://github.com/nvaccess/nvda/pull/16823 also intends to add a profile upgrade. I wonder whether we should aim at having only one config spec increment per release. We can choose several approaches here:

1. Keep config spec version until beta, then raise it. This has the downside that alpha users don't get upgrades from the start
```suggestion
latestSchemaVersion = 11
```
2. Update config spec version as part of this pr, extend the upgrade step in https://github.com/nvaccess/nvda/pull/16823. This has the downside that the additions to the upgrade step will never run for Alpha users as soon as they run a version that contains the new spec version
3. Leave as is, i.e. increment the spec version for every pr rather than for every release
4. Create a new intermediate config spec upgrade branch, rebase this as well as https://github.com/nvaccess/nvda/pull/16823 on that branch, and merge that branch into master as soon as both this pr and https://github.com/nvaccess/nvda/pull/16823 are merged in the new branch.

Reply:
I think option 3 makes the most sense, though is also the most work. @seanbudd, do you have any thoughts?

Reply:
I agree with 3, we need to handle the alpha case where users have upgraded between alpha releases.